elixir/apps/web/lib/web/components/layouts/app.html.heex

```diff
@@ -25,26 +35,18 @@
 
   <.sidebar_item
     current_path={@current_path}
-    navigate={~p"/#{@account}/relay_groups"}
-    icon="hero-arrows-right-left"
-  >
-    Relays
-  </.sidebar_item>
-
-  <.sidebar_item
-    current_path={@current_path}
-    navigate={~p"/#{@account}/sites"}
-    icon="hero-globe-alt"
+    navigate={~p"/#{@account}/policies"}
+    icon="hero-shield-check-solid"
   >
-    Sites
+    Policies
   </.sidebar_item>
 
   <.sidebar_item
     current_path={@current_path}
-    navigate={~p"/#{@account}/policies"}
-    icon="hero-shield-check-solid"
+    navigate={~p"/#{@account}/relay_groups"}
+    icon="hero-arrows-right-left"
   >
-    Policies
+    Relays
```

Comment:
I was thinking we were going to hide Relays for the time being during the beta launch, am I misremembering that?  If so, should we put this behind a feature flag?

Reply:
> When I tried this locally I was redirected to the 'Sites' page on login, but the 'Sites' nav item on the lefthand nav bar was not highlighted.

@bmanifold the breadcrumbs are broken in generally, not because of this PR. They don't work because they expect a slug used but we sometimes redirect to `/:id/sites` which fails here https://github.com/firezone/firezone/blob/andrew/post-login-redirect-to-sites/elixir/apps/web/lib/web/components/navigation_components.ex#L159.

I will push a fix.

> Also, all of the 'Home' links in the breadcrumbs took me to the 'Actors' page.

Thank you! Fixed.

> I was thinking we were going to hide Relays for the time being during the beta launch, am I misremembering that? If so, should we put this behind a feature flag?

Relays should be hidden with a separate PR, here I just reorganized the menu items.